### DISCUSSION TITLE
Exclude snapshots folder

### DISCUSSION THREAD

**Discussion by R2R29** [navidrome/navidrome#3257] [Q&A] [answered]
Hi,
I installed Navidrome on a Synology NAS using Container Manager but it seems to be scanning also my snapshots folder since it's showing artists that I already deleted some time ago.
That snapshot folder is inside my music folder.

Hiw can I exclude the snapshot folder on Container Manager to Navidrome doesn't scan it?

Thanks

**Comment by deluan** [comment]
A simple way is to add an empty `.ndignore` file inside the snapshot folder.

**Reply by R2R29** [reply]
txt file with that extension?

And do I have to stop Navidrome on the Container, or it doesn't matter?

Update: I don't have an option to create a file inside the snapshots folder

Thanks

**Reply by deluan** [reply]
> txt file with that extension?

~~No~~. I mean, it can be any type of file, but it must be named `.ndignore` (starts with a `.` and has not extensions. You'll probably need to create it from a Linux machine/terminal.

> And do I have to stop Navidrome on the Container, or it doesn't matter?

No, just do a rescan after creating the file

> Update: I don't have an option to create a file inside the snapshots folder

Not even using the terminal?
```
$ touch SNAPSHOTFOLDER/.ndignore
```

Another option, which is more complicated and also involves using the terminal is to create another top level folder, outside your Music Folder. Then create symlinks for the folders containing your music into this new top level folder. Then configure your Music Folder to point to this new top level folder.

**Reply by R2R29** [reply]
I'll try using terminal (I'm not very tech savvy but... 😆).

Since I have other folders with snapshots, do I need to specify which one when pasting that on terminal? And how
This is the path: /volume1/music/#snapshot

Sorry for so many questions.

(I went to Jellyfin, Emby, and Plex/Plexamp) and none shows deleted files from the snapshots folder 🤷🏼)

**Reply by deluan** [reply]
So, assuming your Music Folder is `/volume1/music` and the snapshot folder inside it is `/volume1/music/#snapshot`, you open the terminal and try:
```
touch /volume1/music/#snapshot/.ndignore
```

If you have the right permissions, it should create an empty `.ndignore` file inside the snapshot folder.

**Reply by R2R29** [reply]
It doesn't work, even if I log in using root because Snapshots folder is read-only.

Well, unfortunately it seems that I will have to use Jellyfin, Emby or Plex/Plexamp also to my music.

Thanks anyway

**Reply by deluan** [reply, marked as answer]
I'll add `#snapshot` as an ignored folder, so next release (this weekend) will ignore it automatically. You're welcome :)

In a future release (after #2709), I'll add the ability to exclude folders in the UI, without the need to create files in the filesystem.

**Reply by R2R29** [reply]
That would be awesome!

Thanks!!!!!!

**Reply by deluan** [reply]
Done: https://github.com/navidrome/navidrome/commit/9ae2ec1a071d7c07e77363963606ebaffaa796be

Look for a new release next week.

**Reply by R2R29** [reply]
Sorry, only know I saw your last message.

It will be automatically or do we have to change something? Please tell me that the only thing is to rescan 😆

Thanks!!!

**Reply by deluan** [reply]
After the release, you'll just need to upgrade and rescan. A quick rescan will be enough.

**Reply by R2R29** [reply]
@deluan Now it's working flawlessly!

Thanks a lots!
